src/util/errorHandling/errorHandler.ts

```diff
@@ -4,7 +4,7 @@ import { HttpResponseCode } from "./httpResponseCode";
 import { NextFunction } from "connect";
 import { sendEmail } from "../mail";
 
-const toEmails: string[] = ["kzalys@gmail.com"];
+const toEmails: string[] = ["admin@unicsmcr.com"];
```

Comment:
We need to think about moving this kind of hard-coded data to a central settings file

Reply:
There is a lot of these types of settings that we need to move to a settings file. I think that can be done in a separate issue & pull request.

Reply:
Yeah, this was more of a reminder for myself that we still need to do that at some point 😂